package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@exercism/codemirror-lang-wren",
-  "description": "Wren language support for the CodeMirror code editor",
+  "description": "Wren Lezer parser and language support for CodeMirror",
   "author": "Josh Goebel <me@joshgoebel.com>",
-  "version": "0.6.0",
+  "version": "0.7.0",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/exercism/codemirror-lang-wren"
   },
-  "main": "dist/index.cjs",
+  "imports": {
```

Comment:
Hehe. Can you explain what this does? I've read [the docs](https://nodejs.org/api/packages.html#packages_subpath_imports), but I just don't get it 🙈 .

Reply:
It's just a pretty way to define import shortcut/aliases for your own library to use vs having to write out the full path.